src/tabrmd-init.c

```diff
@@ -99,6 +99,12 @@ gmain_data_cleanup (gmain_data_t *data)
     if (data->loop != NULL) {
         main_loop_quit (data->loop);
     }
+    if (data->tpm2 != NULL) {
+        if (data->tpm2->sapi_context != NULL) {
+            g_free(data->tpm2->sapi_context);
+        }
+        g_clear_object (&data->tpm2);
```

Comment:
The `!= NULL` check above seems to say that we should free the pointer, not the address of the pointer, right?

```suggestion
        g_clear_object (data->tpm2);
```